cuppa-surefire/src/main/java/org/forgerock/cuppa/maven/surefire/CuppaSurefireProvider.java

```diff
@@ -26,6 +32,39 @@
      */
     public CuppaSurefireProvider(ProviderParameters parameters) {
         this.providerParameters = parameters;
+        Properties properties = parameters.getProviderProperties();
+        tags = new Tags(getTags(properties), getAntiTags(properties));
+    }
+
+    private Set<String> getTags(Properties properties) {
+        String groups = properties.getProperty("groups");
```

Comment:
Is it not possible to override groups as well? Can I still do `-Dgroups=` on the command line?